src/views/product-tutorials-view/components/sitemap/index.tsx

```diff
@@ -1,34 +1,77 @@
 import Link from 'next/link'
+import classNames from 'classnames'
 import {
   getCollectionSlug,
   getTutorialSlug,
 } from 'views/collection-view/helpers'
+import {
+  Collection as ClientCollection,
+  ProductOption,
+  TutorialLite as ClientTutorialLite,
+} from 'lib/learn-client/types'
 import { ProductPageData } from 'views/product-tutorials-view/server'
+import IconTile from 'components/icon-tile'
+import { IconCollections24 } from '@hashicorp/flight-icons/svg-react/collections-24'
+import { IconExternalLink16 } from '@hashicorp/flight-icons/svg-react/external-link-16'
+import { getSitemapHeading } from 'views/product-tutorials-view/helpers/heading-helpers'
+import StandaloneLink from 'components/standalone-link'
 import s from './sitemap.module.css'
 
 export function ProductTutorialsSitemap({
   collections,
+  product,
 }: {
   collections: ProductPageData['allCollections']
+  product: ProductOption
 }): React.ReactElement {
+  const { title, level, slug } = getSitemapHeading()
+  const HeadingElem = `h${level}` as React.ElementType
+
   return (
-    <ul className={s.root}>
-      {collections.map((collection) => (
-        <li key={collection.id}>
-          <Link href={getCollectionSlug(collection.slug)}>
-            <a>{collection.shortName}</a>
-          </Link>
-          <ul>
-            {collection.tutorials.map((t) => (
-              <li key={t.id}>
-                <Link href={getTutorialSlug(t.slug, collection.slug)}>
-                  <a>{t.name}</a>
-                </Link>
-              </li>
-            ))}
-          </ul>
-        </li>
-      ))}
-    </ul>
+    <div className={s.root}>
+      <div className={s.headingBar}>
+        <HeadingElem id={slug} className={s.heading}>
+          {title}
+        </HeadingElem>
+        <StandaloneLink
```

Comment:
💭 Hmmm... I sort of feel like we should remove this for beta. Feels odd to be booting out back to Learn....But maybe we can just talk with Product / Design as we're doing polish. 

What are your thoughts @zchsh?

Reply:
My memory might be off, but I recall in some Figma sync at some point that we landed on keeping this for June 20 - but maybe with some analytics? The intent being to collect data on how much people would be looking for the "search all tutorials page"?

Fully agreed that it feels a little odd for beta though, with the assumption that I think we want to prioritize collecting feedback on the new experience, so try to keep folks who've opted in within that experience.

Definitely happy to drop this out, or add some `heap` tracking or something - either in this PR or a follow-up 👍

Reply:
Sorry for the confusion, we got feedback from Adam and Geoffrey and decided to remove it all together.

Reply:
Sweet, will do now! 👍

Reply:
Sorry about that I discussed it with Brandon but should have ran it past you two as well. Will be sure to do that in the future!